assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -18,6 +18,13 @@
  * External dependencies
  */
 import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from '@wordpress/element';
```

Comment:
Why do we need `createInterpolateElement` can we load this from React?

Reply:
This is not a React feature. It is specific to `@wordpress/element` and is needed to be able to use HTML markup in translatable strings. See the `Please choose only <b>%s</b> to upload` use case in this file.

Ideally at some point we'd replace it with something custom in order to not rely on this package.

Reply:
As long as you ( @swissspidy ) are happy using this package, then I will mark as resolved.